CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changes
 
+## 0.12.1
```

Comment:
Since we are adding new features, I'll likely publish it as `0.13.0` 🙂 

(I'll change it when publishing, no need to worry about it).